crypto/encode_decode/decoder_meth.c

```diff
@@ -425,6 +430,11 @@ int OSSL_DECODER_number(const OSSL_DECODER *decoder)
     return decoder->base.id;
 }
 
+const char *OSSL_DECODER_name(const OSSL_DECODER *decoder)
```

Comment:
OSSL_DECODER_get_name()

Reply:
**All** of the other functions are `XXX_name()`...